test/runtime/samples/object-rest/main.html

```diff
@@ -0,0 +1,18 @@
+<pre>{JSON.stringify(props)}</pre>
+<script>
+	export default {
+		data: () => ({
+			unwanted: 1,
+			wanted: 2
+		}),
+
+		helpers: {
+			// prevent this being mixed in with data
+			JSON
```

Comment:
Hrm I'm not thrilled with having to do this ... Is it practical to change how whitelisted globals work so that they don't get assigned into the component's data? I actually wasn't aware or had forgotten that this was the case.

Reply:
It's a bit annoying, yeah, but is by far the easiest way to make globals available. The alternative would be to more cleanly separate `state` and the root `ctx` object:

```js
// on init
const base = { JSON };

// on each change
const ctx = assign({}, base, newState);
this._recompute(changed, ctx);
```

That might be necessary anyway at some point, to work around some other quirks relating to merging store with non-store data, and also `this.get({ computed: false })` etc. But I think it's orthogonal to this issue